src/pages/[...blog].astro

```diff
@@ -19,59 +19,65 @@ const { Content } = Astro.props
 <Content />
 
 <script>
-const footnoteEl = document.querySelector('section.footnotes')
+window.addEventListener('DOMContentLoaded', () => {
+  setTimeout(prepFootnotes, 100)
+})
```

Comment:
The footnote related fix merged in PR #90 somehow breaks in any blog posts that are turned into `.mdx` files. But giving the logic a reasonable amount of delay makes it work in both `.md` and `.mdx` files. (Let me know if this needs to be commented within the file itself)

Reply:
Yeah, all unusual or odd or weird behavior needs comments 👍